### DISCUSSION TITLE
When is SDK 1.11 expected to be released?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#804] [Q&A] [answered]
Do you have a preliminary release plan for SDK 1.11?

Best regards, Åsa

**Comment by YvesJo** [comment, marked as answer]
Hi,
It could be expected for end of February. 
KR

**Reply by klakegg** [reply]
Is this still on for next week?

**Reply by YvesJo** [reply]
Hi,
It won't. The release is also dependent from factors outside the SDK, so for more reliable schedules, I invite you to preferably get in touch directly with the helpdesk: ted@publications.europa.eu
KR

**Reply by klakegg** [reply]
Why can't a reliable schedule be available to all the users of the SDK without getting in touch by email?

**Reply by YvesJo** [reply]
Hi,
The Release Candidate should be available early next week, which will give you the opportunity to see what the release will contain. From there, about 2 weeks are needed to perform the final controls before releasing SDK 1.11.

**Reply by Colan71** [reply]
Any news about release date for SDK 1.11?

**Reply by YvesJo** [reply]
So far, Release Candidate 3 looks fine and the release could be ready next week. A non-SDK related technical concern has to be addressed before its use in production and we expect to have it done shortly.
